exp/web-dashboard.md

```diff
@@ -0,0 +1,8 @@
+The Anbox Cloud dashboard offers a web interface to create, manage and stream applications. The dashboard is especially helpful for developers new to Anbox Cloud and helps them by offering a simpler and intuitive management interface.
+
+The dashboard comes pre-installed when you deploy the full version of Anbox Cloud (along with the streaming stack) or the Anbox Cloud appliance. It operates from behind a reverse proxy for performance and security reasons. The dashboard relies on OAuth for user authentication. The only OAuth provider supported right now is [Ubuntu One](https://login.ubuntu.com/). Future versions of Anbox Cloud will allow using other OAuth providers.
```

Comment:
`Future versions of Anbox Cloud will allow using other OAuth providers.` 

I know the dashboard will implement oidc & oauth support but just to confirm, is this something that we should put out publicly and will it be the same `other` providers ?

Reply:
Dropped that statement entirely as per Gary's suggestion below.